diff --git a/Makefile b/Makefile
index 7c2ac1f..f5bfbbe 100644
--- a/Makefile
+++ b/Makefile
@@ -146,7 +146,7 @@ other:
 	@echo -e "\n* Copying over Makefile to chroot ..."
 	@cp Makefile config.mk /mnt
 	@echo -e "\n* Changing root to system ..."
-	@arch-chroot /mnt make $(ARCH_BASE)
+	@arch-chroot /mnt
 
 ## Run this command when your done with all other commands.
 
diff --git a/config.mk b/config.mk
index 5569e55..8387ff5 100644
--- a/config.mk
+++ b/config.mk
@@ -81,9 +81,6 @@ BOOT_OPTS := quiet loglevel=3 systemd.show_status=false rd.udev.log_level=3 vt.g
 # Define initramfs base system options:
 INITRAMFS_OPTS := base systemd autodetect microcode modconf kms keyboard sd-vconsole block filesystems
 
-# Define Arch Linux base system make operations:
-ARCH_BASE := archlinux-system archlinux-silent archlinux-dev
-
 # Define base system packages.
 # (This includes the kernel and kernel firmware.)
 PKGS_BASE := base $(KERNEL) $(MICROCODE) $(BOOTLOADER)
